pkg/machine/e2e/config_windows_test.go

```diff
@@ -3,14 +3,39 @@ package e2e_test
 import (
 	"fmt"
 	"os/exec"
+	"path/filepath"
 	"strings"
 
+	"github.com/containers/libhvee/pkg/hypervctl"
+	"github.com/containers/podman/v6/pkg/machine/define"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega/gexec"
 )
 
 const podmanBinary = "../../../bin/windows/podman.exe"
 
+var (
+	// fakeImagePath string = "C:\\Users\\Baude\\fakedisk.vhdx"
+	fakeImagePath string = ""
+)
+
+func initPlatform() {
+	switch testProvider.VMType().String() {
+	case define.HyperVVirt.String():
+		vmm := hypervctl.NewVirtualMachineManager()
+		fullFileName := filepath.Join(tmpDir, randomString()) + ".vhdx"
```

Comment:
nit: can you name that `"podman-fakeimage"+randomString()` so it is more clear when looking at the fs where this comes from.

Also this is using the tmpDir which seems to be the global OS tmpdir so we leak that forever AFAICT. I think this would need to get removed in SynchronizedAfterSuite() then to avoid the leaks.

Reply:
done